pages/pipelines/notifications.md.erb

```diff
@@ -80,9 +80,15 @@ The `basecamp_campfire` attribute accepts a single URL as a string.
 
 ## Slack Channel Message
 
-To send notifications to a Slack Channel, first set up the Slack notification service in your Organization's notification settings. Detailed instructions for setting up a Slack notification service can be found in the [Slack integration guide](/docs/integrations/slack). 
+Many teams find the messages generated by the [Slack integration configured with the buildkite.com web interface](/docs/integrations/slack) meet their needs. However, it's also possible to configure slack notifation within pipeline YAML to gain some extra configuration options.
```

Comment:
“Many teams find the messages generated by the Slack integration configured with the buildkite.com web interface meet their needs” reads a bit overly defensively to me.

I’m not sure what to suggest more broadly, but a few wording tweaks for the rest:

```suggestion
Many teams find the messages generated by a [Slack notification service configured from the Buildkite web interface](/docs/integrations/slack) meet their needs. However, it's also possible to configure Slack notifations from YAML Pipelines for more control.
```

Reply:
How about 

> If your need more control over your Slack notifications than the [Buildkite web interface](/docs/integrations/slack) provides, you can control them from your YAML Pipelines.